src/controllers/admin_room.js

```diff
@@ -0,0 +1,30 @@
+import models from '../database/models';
+
+export default (req, res, next) => {
+  const { params: { id: roomId } } = req;
+  let roomName = '';
```

Comment:
no need for this line here 
you still can access the  variable 
delete this line 
and change line 7 to `let   roomName = roomInfo.room_name;`